### PR TITLE
Don't load JRequest when it isn't used.

### PR DESCRIPTION
**Pull request by realityking**

### CODE REVIEW

File: libraries/import.php [hunk 1]
```diff
@@ -67,7 +66,7 @@ class_exists('JLoader') or die;
  */
 if (isset($_SERVER['HTTP_HOST']))
 {
-	JLoader::import('joomla.environment.request');
+	JLoader::register('JRequest', JPATH_PLATFORM . '/joomla/environment/request.php')
```
Comment (elkuku) [inline libraries/import.php]: Missing semicolons produce nice parse errors ;)

Comment (realityking) [inline libraries/import.php]: Gosh darn. Should be fixed now.

**Comment by elinw** [comment]
Rouven i know that in my environment there are places where tests fail because JRequest is not loaded even though they pass the tests that jenkins runs, so I would be extremely cautious about this.

**Comment by realityking** [comment]
Unit tests don't go trough import.php, so this shouldn't affect unit tests.

The unit test with JRequest issues probably need a require_once to fix them.

**Pull request merged**